compiler/one-cmds/one-import-bcq

```diff
@@ -14,219 +19,206 @@
-
-# activate python virtual environment
-VIRTUALENV_LINUX="${DRIVER_PATH}/venv/bin/activate"
-VIRTUALENV_WINDOWS="${DRIVER_PATH}/venv/Scripts/activate"
-
-if [ -e ${VIRTUALENV_LINUX} ]; then
-  source ${VIRTUALENV_LINUX}
-elif [ -e ${VIRTUALENV_WINDOWS} ]; then
-  source ${VIRTUALENV_WINDOWS}
-else
-  echo "Error: Virtual environment not found. Please run 'one-prepare-venv' command."
-  echo "If encounter any problems, please follow provided document in 'doc' folder."
-  exit 255
-fi
-
-# remove previous log
-rm -rf "${OUTPUT_PATH}.log"
-
-show_err_onexit()
-{
-  cat "${OUTPUT_PATH}.log"
-}
-
-trap show_err_onexit ERR
-
-# Generate BCQ information metadata
-# If model has no BCQ information or invalid information, pb file is not changed.
-METAGEN_SCRIPT="${DRIVER_PATH}/generate_bcq_metadata "
-METAGEN_SCRIPT+="--input_path ${INPUT_PATH} "
-METAGEN_SCRIPT+="--output_path ${TMPDIR}/${MODEL_NAME}_withmeta.pb "
-METAGEN_SCRIPT+="--output_arrays ${OUTPUT_ARRAYS} "
-
-echo ${METAGEN_SCRIPT} > "${OUTPUT_PATH}.log"
-echo "" >> "${OUTPUT_PATH}.log"
-$METAGEN_SCRIPT >> "${OUTPUT_PATH}.log" 2>&1
-
-# Generate BCQ information nodes as output_arrays
-# If model has no BCQ information, output_arrays would be empty.
-OUTARR_GEN_SCRIPT="${DRIVER_PATH}/generate_bcq_output_arrays "
-OUTARR_GEN_SCRIPT+="--input_path ${TMPDIR}/${MODEL_NAME}_withmeta.pb "
-OUTARR_GEN_SCRIPT+="--metadata_path ${TMPDIR}/${MODEL_NAME}_metadata_arrays.txt "
-OUTARR_GEN_SCRIPT+="--output_arrays_path ${TMPDIR}/${MODEL_NAME}_output_arrays.txt "
-
-echo ${OUTARR_GEN_SCRIPT} >> "${OUTPUT_PATH}.log"
-echo "" >> "${OUTPUT_PATH}.log"
-$OUTARR_GEN_SCRIPT >> "${OUTPUT_PATH}.log" 2>&1
-
-# generate tflite file
-CONVERT_SCRIPT="python ${DRIVER_PATH}/tf2tfliteV2.py ${TF_INTERFACE} "
-CONVERT_SCRIPT+="--input_path ${TMPDIR}/${MODEL_NAME}_withmeta.pb "
-CONVERT_SCRIPT+="--input_arrays ${INPUT_ARRAYS} "
-CONVERT_SCRIPT+="--output_path ${TMPDIR}/${MODEL_NAME}.tflite "
-CONVERT_SCRIPT+="--output_arrays "
-CONVERT_SCRIPT+="$(cat ${TMPDIR}/${MODEL_NAME}_metadata_arrays.txt)"
-CONVERT_SCRIPT+="${OUTPUT_ARRAYS}"
-CONVERT_SCRIPT+="$(cat ${TMPDIR}/${MODEL_NAME}_output_arrays.txt) "
-if [ ! -z ${INPUT_SHAPES} ]; then
-  CONVERT_SCRIPT+="--input_shapes ${INPUT_SHAPES} "
-fi
-
-echo ${CONVERT_SCRIPT} >> "${OUTPUT_PATH}.log"
-$CONVERT_SCRIPT >> "${OUTPUT_PATH}.log" 2>&1
-
-# convert .tflite to .circle
-echo " " >> "${OUTPUT_PATH}.log"
-echo "${DRIVER_PATH}/tflite2circle" "${TMPDIR}/${MODEL_NAME}.tflite" \
-"${OUTPUT_PATH}" >> "${OUTPUT_PATH}.log"
-echo " " >> "${OUTPUT_PATH}.log"
-
-"${DRIVER_PATH}/tflite2circle" "${TMPDIR}/${MODEL_NAME}.tflite" \
-"${OUTPUT_PATH}" >> "${OUTPUT_PATH}.log" 2>&1
+import argparse
+import os
+import subprocess
+import sys
+import tempfile
+
+import utils as _utils
+import generate_bcq_output_arrays as _bcq_info_gen
+
+
+def _get_parser():
+    parser = argparse.ArgumentParser(
+        description='command line tool to convert TensorFlow with BCQ to circle')
```

Comment:
`command ` -> `Command`

Reply:
Actually, this is intended. `argparse` python library print default help message as small letter. So, I matched it.